tests/CMakeLists.txt

```diff
@@ -108,6 +108,8 @@ function(set_props target)
   # DrMem itself gets ARM from dr_api.h but tests need it on their own:
```

Comment:
> aarch64-inital-tests-port

Please follow the conventions on commit titles: https://dynamorio.org/page_code_reviews.html#sec_commit_messages

Is this part of #2016?  Please include that in the title and the body.

I would expect something like `i#2016 aarch64, part 1: Enable pattern mode tests`